internal/dkg/execution.go

```diff
@@ -217,7 +198,7 @@ func (d *Process) startDKGExecution(
 			finalGroup = append(finalGroup, config.NewNodes[v.Index])
 		}
 
-		groupFile, err := asGroup(ctx, current, &share, finalGroup)
+		groupFile, err := asGroup(ctx, current, &share, finalGroup, transitionTime)
```

Comment:
You could take the reference on line 193 directly too to avoid taking the ref here, but it's nitpick